sr_gui_controller_tuner/src/sr_gui_controller_tuner/controller_tuner.py

```diff
@@ -498,53 +537,56 @@ def refresh_controller_tree_(self, controller_type = "Motor Force"):
                         frame_buttons = QFrame()
                         layout_buttons = QHBoxLayout()
                         btn_plot = QPushButton("Plot")
-                        self.ctrl_widgets[ motor_name ]["btn_plot"] = btn_plot
-                        self.ctrl_widgets[ motor_name ]["btn_plot"].clicked.connect(partial(self.on_btn_plot_pressed_, motor_name, self.ctrl_widgets[ motor_name ]["btn_plot"]))
+                        self.ctrl_widgets[motor_name]["btn_plot"] = btn_plot
+                        self.ctrl_widgets[motor_name]["btn_plot"].clicked.connect(partial(self.on_btn_plot_pressed_,
+                                                                                  motor_name, self.ctrl_widgets[motor_name]["btn_plot"]))
                         layout_buttons.addWidget(btn_plot)
 
                         if self.controller_type in ["Position", "Muscle Position", "Mixed Position/Velocity"]:
                             #only adding Move button for position controllers
                             btn_move = QPushButton("Move")
-                            self.ctrl_widgets[ motor_name ]["btn_move"] = btn_move
-                            self.ctrl_widgets[ motor_name ]["btn_move"].clicked.connect(partial(self.on_btn_move_pressed_,motor_name, self.ctrl_widgets[ motor_name ]["btn_move"]))
+                            self.ctrl_widgets[motor_name]["btn_move"] = btn_move
+                            self.ctrl_widgets[motor_name]["btn_move"].clicked.connect(partial(self.on_btn_move_pressed_, motor_name,
+                                                                                      self.ctrl_widgets[motor_name]["btn_move"]))
                             layout_buttons.addWidget(btn_move)
                             frame_buttons.setLayout(layout_buttons)
 
-                        self._widget.tree_ctrl_settings.setItemWidget(  motor_item, 0, frame_buttons )
+                        self._widget.tree_ctrl_settings.setItemWidget(motor_item, 0, frame_buttons)
 
-                    for index_item,item in enumerate(ctrl_settings.headers):
+                    for index_item, item in enumerate(ctrl_settings.headers):
                         if item["type"] == "Bool":
                             check_box = QCheckBox()
 
-                            if parameter_values["sign"] == 1.0:
-                                check_box.setChecked(True)
-
-                            check_box.setToolTip("Check if you want a negative sign\n(if the motor is being driven\n the wrong way around).")
-                            self._widget.tree_ctrl_settings.setItemWidget(  motor_item, index_item, check_box )
+                            param_name = item["name"].lower()
+                            param_val = parameter_values[param_name]
+                            check_box.setChecked(param_val)
+                            if "sign" in param_name:
```

Comment:
The behavior before change was that all parameter called sign would be read and if value was 1 it would check the checkbox. This means any other names boolean would never be read.
The tooltip was always set to this checkbox, and only concerns parameters named sign. 
Hence, if the parameter name contains sign (and not is equal otherwise we might miss velocity/sign), the tooltip related to sign is added.